bambi/plots/__init__.py

```diff
@@ -1,3 +1,16 @@
-from .plot_cap import create_cap_data, plot_cap
+from bambi.plots.create_data import create_cap_data, create_comparisons_data
+from bambi.plots.effects import comparisons, predictions
+from bambi.plots.plotting import plot_cap, plot_comparison
+from bambi.plots.plot_types import plot_numeric, plot_categoric
 
-__all__ = ["create_cap_data", "plot_cap"]
+
+__all__ = [
+    "create_cap_data",
+    "create_comparisons_data",
+    "comparisons",
+    "predictions",
+    "plot_cap",
+    "plot_comparison",
+    "plot_numeric",
+    "plot_categoric",
```

Comment:
Do you think users will make use of these functions? Similar for `create_cap_data` and `create_comparisons_data`

Reply:
No, they shouldn't. Users should only use `plot_*` along with `predictions` & `comparisons`.